### PR TITLE
Refine alias updating instructions in README

### PR DESCRIPTION
**Pull request by refack**
Refs: https://github.com/nodejs/email/issues/135

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,32 +5,34 @@
...
 
 
 ## Updating
 
 ### Responsibility
 
-Merging a PR sould be done concomitantly with updating the mail routes in Mailgun. Since access to the Mailgun API key is required, only members of the @nodejs/build-infra team have the permission to push code to the `master` branch. That is in order to minimize the probability of mismatches between the information in this repo, and the actual email routes setup.
-
-The Mailgun API key can be found in the `admin_logins.md` file in the secrets repo (build/infra/), or direclty via the Rackspace API.
+Changes to the information in this repo sould be done concomitantly with updates to the configured mail routes in Mailgun. Since access to the Mailgun API key is required, only members of the @nodejs/build-infra team have the permission to push code to the `master` branch. That was done in order to minimize the posibility of mismatches between the information in this repo, and the actual email routes that are set up.
```
Comment (rvagg) [inline README.md]: I think the first sentence might make this paragraph less clear than if it weren't there. Perhaps remove it

File: README.md [hunk 2]
```diff
@@ -5,32 +5,34 @@
...
+The [update](./update) directory contains a node program which will read the aliases mapping file, fetch the list of mail routes from Mailgun and update the routes to make sure they match the required state. The program is run by passing it a domain name as an agument (`update/update.js iojs.org`).
+
+The Mailgun API key for the given domain is required. It can be found in the `admin_logins.md` file in the secrets repo (build/infra/), or direclty via the Rackspace API. The key should be stored as a file named `iojs.org/credentials.json` in the form: `{ "api-key": "key-abc..." }` within a code tree.
+
+The programs can be used with `--dry-run` to verifry the current status and what will be changed by an update.
```
Comment (rvagg) [inline README.md]: *verify

File: README.md (hunk 1 above)
Comment (refack) [reply README.md]: Ack

File: README.md (hunk 2 above)
Comment (refack) [reply README.md]: Ack

File: README.md [hunk 3]
```diff
@@ -2,26 +2,39 @@
...
+# nodejs.org
 
-* create an iojs.org alias and associate the intended recipients to it.
-* email 'helpdesk@rt.linuxfoundation.org' requesting that the nodejs.org point to the alias. In the email, CC a Node.js Foundation contact (Executive Director or other staff) and provide context as to why the alias is needed.
+`@nodejs.org` email adresses are managed by the Linux Foundation. In order to set one up:
+* Create an iojs.org alias with associated recipients as described above.
+* Send an email to 'helpdesk@rt.linuxfoundation.org' requesting that the wanted nodejs.org address by directed to the new alias. Please CC an OpenJS Foundation contact person and provide context as to why the alias is needed.
 
-## License & copyright
+#### License & copyright
```
Comment (rvagg) [inline README.md]: can you remove these extra `##`'s please @refack?

File: README.md [hunk 4]
```diff
@@ -2,26 +2,39 @@
...
+
+The Mailgun API key for the given domain is required. It can be found in the `admin_logins.md` file in the secrets repo (build/infra/), or direclty via the Rackspace API. The key should be stored as a file named `iojs.org/credentials.json` in the form: `{ "api-key": "key-abc..." }` within a code tree.
 
-Members of the Build Working Group "Infra" team (Rod Vagg, Johan Bergström, Michael Dawson and João Reis as of writing) can access the Mailgun API key via the Rackspace API or in the `admin_logins.md` file in the secrets repo (build/infra/). This key must be placed in a file named `iojs.org/credentials.json` in the form: `{ "api-key": "key-abc..." }`.
+The programs can be used with `--dry-run` to verify the current status and what will be changed by an update.
 
-## nodejs.org email aliases
 
-nodejs.org aliases are managed by the Linux Foundation.  In order to set one up:
+# nodejs.org
```
Comment (rvagg) [inline README.md]: this should get an extra `#`

File: README.md (hunk 3 above)
Comment (refack) [reply README.md]: done.

File: README.md (hunk 4 above)
Comment (refack) [reply README.md]: and done

**Comment by refack** [comment]
@Trott would you mind TAL?

**Pull request merged**